lib/user_agent/browsers/internet_explorer.rb

```diff
@@ -6,27 +6,27 @@ class InternetExplorer < Base
         "Trident/7.0" => "11.0",
         "Trident/6.0" => "10.0",
         "Trident/5.0" => "9.0",
-        "Trident/4.0" => "8.0",
```

Comment:
I would allow or even enforce trailing commas as it makes for clearer diffs in the future when a new value is appended to an array like this.

Reply:
It's a hash in this case, either way trailing commas are a good thing for both arrays and hashes in my experience.

Reply:
addressed for both hashes and arrays